### ISSUE TITLE
Invalid pagination querystring being produced when pagination_type is :page (`limit` instead of `page_size`) 

### ISSUE THREAD

**Issue by jesse-iluminai** [woylie/flop_phoenix#20]
Hi Woylie,

I noticed that with the newest release here of 0.6.0, when I'm working with ":page" style pagination (page/page_size) I'm getting errors with the link builder helper in the UI where it is producing a URL of "http://localhost:4000/berries?page=2&limit=15".

On the previous release when I had `default_limit` and `max_limit` set to 15 each `limit` in the URL above would resolve to the correct `page_size` param as in "http://localhost:4000/berries?page=2&page_size=15"

This is with both Flop 0.10.0 and 0.9.1.  Confirmed that the issue does not exist with flop_phoenix 0.5.0.

Here's what i have in my config.exs
```
config :flop,
  repo: App.Repo,
  default_limit: 15,
  pagination_types: [:page]
```

And here's what's being produced when I inspect flop.meta within the build_page_link_helper function:

```
%Flop{
  after: nil,
  before: nil,
  filters: [],
  first: nil,
  last: nil,
  limit: 15,
  offset: nil,
  order_by: nil,
  order_directions: nil,
  page: nil,
  page_size: nil
}
```

Any thoughts here?

Cheers,
Jesse

**Comment by woylie** [comment]
Thanks for the bug report, I just released 0.6.1 with a fix.

**Comment by jesse-iluminai** [comment]
Awesome - thanks for the turnaround on this one Mathias
